flocker/provision/_ssh/_model.py

```diff
@@ -52,15 +52,20 @@ class Run(PRecord):
     :ivar bytes command: The command to run.
     :ivar callable log_command_filter: A filter to apply to any logging
         of the executed command.
+    :ivar bytes pre_command: A command to run before the ``command``,
+        generally to setup the environment for the command.
```

Comment:
Oh right, because `virtualenv` weirdly doesn't provide a way of running arbitrary commands inside a virtualenv. Even so, its cleaner to change `command` to `commands`.